src/Components/Components/src/NavigationManager.cs

```diff
@@ -420,10 +427,10 @@ protected virtual void SetNavigationLockState(bool value)
         => throw new NotSupportedException($"To support navigation locks, {GetType().Name} must override {nameof(SetNavigationLockState)}");
 
     /// <summary>
-    /// Adds a handler to process incoming navigation events.
+    /// Registers a handler to process incoming navigation events.
     /// </summary>
     /// <param name="locationChangingHandler">The handler to process incoming navigation events.</param>
```

Comment:
Technically there should be a `<returns>...</returns>` part of these XML docs now.